library/core/src/net/ip_addr.rs

```diff
@@ -468,7 +468,13 @@ impl Ipv4Addr {
     #[unstable(feature = "ip_bits", issue = "113744")]
     pub const BITS: u32 = 32;
 
-    /// Converts an IPv4 address into host byte order `u32`.
+    /// Converts an IPv4 address into a `u32` representation using native byte order.
+    ///
+    /// Although IPv4 addresses are big-endian, the `u32` value will use the target platform's
+    /// native byte order. That is, the `u32` value is an integer representation of the IPv4
+    /// address and not an integer interpretation of the IPv4 address's big-endian bitstring. This
+    /// means that the `u32` value `&` with `0xffffff00` will set the last octet in the address to
```

Comment:
Using the `&` operator as a verb seems confusing to me -- maybe we can leave that to the example and use "masked" here?
```suggestion
    /// means that the `u32` value masked with `0xffffff00` will set the last octet in the address to
```

Reply:
That sounds a lot better, thanks. Update pushed.